sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 2.0.0-beta.2 (Unreleased)
 
+- For the `InteractiveBrowserCredential`, we're dropping support for [Implicit Grant Flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) to use only [Auth Code Flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow).
```

Comment:
I would make this more direct and remove implementation details. For your consideration:

- `InteractiveBrowserCredential` no longer supports Implicit Grant Flow. The implicit flow will not work in modern browsers which block third-party cookies. Because `InteractiveBrowserCredential` only supports Auth Code Flow now, the `flow` option introduced in `1.2.4-beta.1` has been removed.

Reply:
Thank you! I'll take it